src/resolver.js

```diff
@@ -12,11 +12,6 @@ const util = require('./util')
  *
  * @param {Uint8Array} binaryBlob - Binary representation of a Bitcoin block
  * @param {string} [path='/'] - Path that should be resolved
- * @returns {Object} result - Result of the path it it was resolved successfully
```

Comment:
I know that TypeScript automatically fills in return values. But is there anyway to still keep the information about what the return values are about in the docs somehow? Perhaps as a section before the types?